.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,60 @@
+name: Publish cpu dockers
+on:
+  workflow_dispatch
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: TissueImageAnalytics/tiatoolbox
+  image: ghcr.io/tissueimageanalytics/tiatoolbox
+  TOOLBOX_VER: 1.3.0
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - dockerfile: ./3.7/Debian/Dockerfile
+            mtag: py3.7-debian
+          - dockerfile: ./3.8/Debian/Dockerfile
+            mtag: py3.8-debian
+          - dockerfile: ./3.9/Debian/Dockerfile
+            mtag: py3.9-debian
+          - dockerfile: ./3.9/Ubuntu/Dockerfile
+            mtag: py3.9-ubuntu
+          - dockerfile: ./3.9/Ubuntu/Dockerfile
+            mtag: latest
```

Comment:
How about python 3.10?

Reply:
Sure, I will add it.

Reply:
Done in [7f030d8](https://github.com/TissueImageAnalytics/tiatoolbox/pull/504/commits/7f030d821f83514aad5385703be701e856f583a4)